src/models/SecretarySettings/Errors/SecretarySettingsNotFound.ts

```diff
@@ -0,0 +1,11 @@
+import { Secretary } from "$src/models/Admin";
+
+export class SecretarySettingsNotFoundError extends Error {
```

Comment:
lo cambiaria por SecretarySetting, para poder distinguir bien singular para una instancia y la tabla

abajo tmb podriamos poner en vez de "The SecretarySettings for the secretary of", "The ${secretary} secretary setting doesn't exist"

Reply:
Es que para mi cada campo en la tabla salvo el enum de secretarias es un setting distinto. Por eso el plural me parece q es la que va